openlayers/openlayers.d.ts

```diff
@@ -203,7 +203,7 @@ declare module ol {
          * @template S
          * @api stable
          */
-        forEach<S>(f: ((self: S, item: T, index: number, array: T[]) => any), opt_this?: S): void;
+        forEach<S, T>(f: ((item: T, index: number, array: T[]) => any), opt_this?: S): void;
```

Comment:
I don't think 'T' is necessary here since it is defined in the class declaration.